docs/using.rst

```diff
@@ -188,6 +190,24 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant 2 way folder sync
```

Comment:
@hferentschik I have used "2 way folder sync" because that's the term mostly used in articles, forums if you search internet about the topic. However I dont have any issue using "bi-directional synced folder " as this is used Vagrant documents. Refer : https://www.vagrantup.com/docs/synced-folders/smb.html